canonical/snapcraft.io
# Remove new table feature flag

## Done
- Removed the feature flag for the new snaps tables

## How to QA
- Go to https://snapcraft-io-4527.demos.haus/admin
- Check that you get the latest snaps view
- Check that the snaps view works as expected

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-8818
